pkg/common/api_errors.go

```diff
@@ -0,0 +1,62 @@
+package common
+
+const (
+	// Http
+	ErrMalformedRequest     = "failed deserializing HTTP body: %s"
```

Comment:
Are these constant shared among packages? `common.ErrMalformedRequest` is very generic name, but it seems to be HTTP specific here.

Also, I am not very sure about moving string templates away from the call site of `fmt.Sprintf` -- it becomes a lot harder for the reader to see if we are passing the right number of args, or passing the correct arg type. Previously it means reading a single line to determine, now it is another line in another file in a different package.

One way to assist is to turn these info `func` instead of `string` and relying on compiler type checking.

I'll leave it to @yaron2 and @youngbupark to make the call here.

Reply:
these constants are shared between packages of 'grpc' and 'http',  however, some constants are delegated to 'http' package only.

Reply:
I'm not sure. @yaron2 @youngbupark WDYT ^

Reply:
can someone give more comments? this PR has been pending for days.

Reply:
Sorry for the delay, @beiwei30. Lets do as you suggested and change "common" into more specific names.

Reply:
@yaron2 I've changed package name to 'messages', pls. check.